addons/l10n_latam_check/models/account_chart_template.py

```diff
@@ -30,11 +33,26 @@ def _get_latam_check_account_journal(self, template_code):
                     'name': _('Rejected Third Party Checks'),
                     'type': 'cash',
                     'outbound_payment_method_line_ids': [
-                        Command.create({'payment_method_id': self.env.ref('l10n_latam_check.account_payment_method_out_third_party_checks').id}),
+                        Command.create({
+                            'payment_method_id': self.env.ref('l10n_latam_check.account_payment_method_out_third_party_checks').id,
+                            'payment_account_id': 'outstanding_check_out',
+                        }),
                     ],
                     'inbound_payment_method_line_ids': [
                         Command.create({'payment_method_id': self.env.ref('l10n_latam_check.account_payment_method_new_third_party_checks').id}),
                         Command.create({'payment_method_id': self.env.ref('l10n_latam_check.account_payment_method_in_third_party_checks').id}),
                     ],
                 },
             }
+
+    @template(model='account.account')
+    def _get_latam_check_outstanding_account_account(self, template_code):
```

Comment:
hoooooo that's clever :) I suppose you can ignore one of my previous comment about the lost of the outstanding accounts in the chart templates :-)

nice one nice one :+1: